geodiff/CMakeLists.txt

```diff
@@ -14,7 +14,6 @@ SET(ENABLE_TESTS TRUE CACHE BOOL "Build tests?")
 SET(ENABLE_COVERAGE FALSE CACHE BOOL "Enable GCOV code coverage?")
 SET(BUILD_TOOLS TRUE CACHE BOOL "Build tool executables?")
 SET(BUILD_STATIC FALSE CACHE BOOL "Build static libraries?")
-SET(WITH_INTERNAL_SQLITE3 TRUE CACHE BOOL "Use internal copy of SQLITE3")
```

Comment:
can you do instead a force ERROR message 
like 

```
if (WITH_INTERNAL_SQLITE3)
  message(ERROR this option was removed in geodiff 1.x due to the ....)
endif()

Reply:
added in e148516